app/en/home/build-tools/create-a-tool-with-auth/page.mdx

```diff
@@ -116,12 +118,10 @@ async def get_posts_in_subreddit(
 To use this tool, you need to install the Arcade CLI and run 'arcade login' to authenticate:
 
 ```bash
-uv pip install arcade-mcp
+uv tool install arcade-mcp
 arcade login
 ```
 
-Or add your `ARCADE_API_KEY` and `ARCADE_USER_ID` as environment variables or an `.env` file at the root of your project.
-
```

Comment:
I want them to login. Also, using env var or .env is finicky when dealing with stdio transport

Reply:
I agree with this, but I've also found it slightly confusing that if I need a tool with a secret I have to use the `.env` file anyway, especially if I run `arcade configure` where (with stdio):
- I can't get it to reliably inject the Arcade API key
- If I have an .env file, it will load _only_ those variables, and omit the Arcade API key.

So it sound to me that we need to clarify here that the `.env` route is valid (and for secrets maybe required)

Reply:
I'm not willing to die on this hill haha, so if the following two points don't convince you, then I can revert. Let me know.

1. I'm not convinced that the MCP server developer needs to know that under the hood we use their ARCADE_API_KEY and ARCADE_USER_ID. They just need to know that `arcade login` authenticates them.
2. `arcade login` links their Arcade account with CLI usage & 'MCP server started' events sent to PostHog.